tools/dwp_tools.py

```diff
@@ -1,8 +1,9 @@
 from shapely.geometry import shape, Point, LineString, MultiLineString
+import arcpy
```

Comment:
arcpy niet in gistools implementeren. Mochten we de functies dan ergens anders nodig hebben (bijvoorbeeld een qgis plugin), dan moeten we altijd een arcgis licentie hebben (en het werkt inderdaad ook niet in travis).
Arcpy wordt hier alleen voor logging gebruikt. 
Voor logging heb ik in mijn branch van de TijhuisArdGisTools een oplossing zitten. Hier gewoon toevoegen:
import logging
log = logging.get_logger(__file__)
en het zou moeten werken